File: python/cart_item.py
class CartItem:
    def __init__(self, product, quantity):
        self.product = product
        self.quantity = quantity
    
    def get_product(self):
        return self.product
    
    def get_quantity(self):
        return self.quantity
    
    def get_total_price(self):
        return self.product.get_price() * self.quantity
    
    def set_quantity(self, quantity):
        self.quantity = quantity

File: csharp/CartItem.cs
// csharp/CartItem.cs
namespace ECommerce
{
    public class CartItem
    {
        public Product Product { get; private set; }
        public int Quantity { get; set; }

        public CartItem(Product product, int quantity)
        {
            Product = product;
            Quantity = quantity;
        }

        public double GetTotalPrice()
        {
            return Product.Price * Quantity;
        }
    }
}
